tests/columns/test_ip.py

```diff
@@ -0,0 +1,194 @@
+from __future__ import unicode_literals
+
+from tests.testcase import BaseTestCase
+from clickhouse_driver import errors
+from ipaddress import IPv6Address, IPv4Address
+
+
+class IPv4TestCase(BaseTestCase):
```

Comment:
Feel free to add new ClickHouse release with IP addresses support to travis matrix: https://github.com/mymarilyn/clickhouse-driver/blob/master/.travis.yml#L2

Reply:
To skip tests for versions that do not support IP addresses use `require_server_version` decorator: https://github.com/mymarilyn/clickhouse-driver/blob/master/tests/columns/test_decimal.py#L12.

Reply:
Check travis for build status: https://travis-ci.org/mymarilyn/clickhouse-driver/pull_requests

Reply:
I've just pushed branch https://github.com/mymarilyn/clickhouse-driver/tree/feature-version-dependent-settings-in-test where Decimal tests run without errors with latest server version. You can use it as branchpoint for making all tests green.

Reply:
I've added the decorator on all tests, this should work now. Note that tests for the Decimal column will fail on 19.3.3 because the --allow_experimental_decimal_type flag was removed in 18.14.9 (it is enabled by default now), so we may want to add an upper version check decorator